client/lib/paths/login/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import config, { isEnabled } from 'config';
+
+export function login( { redirectTo } ) {
+	const url = isEnabled( 'wp-login' ) ? '/login' : config( 'login_url' );
```

Comment:
> There are few occurrences of config( 'login_url' ) that have not been changed to use the new login() helper (e.g. the Login block component). Is it on purpose?

Yes, they are used in the different contexts, to create account recovery urls for instance. That's why it makes this process more complicated.